### ISSUE TITLE
login.html

### ISSUE THREAD

**Issue by H61cker** [H61cker/-#3]
<html>
<head>
    <meta charset="utf-8">
    <title>login</title>

    <meta name="apple-mobile-web-app-capable" content="yes"/>
    <meta name="viewport" content="width=device-width,initial-scale=1, user-scalable=no"/>
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
    <link rel='stylesheet' href="include/css/bootstrap.min.css"/>
    <style>
        .margin_top { margin-top: 80px; }
    </style>
</head>
<body>

<div class="container">

    <p class="margin_top"></p>

    <div class="jumbotron">
        <div class="container text-center">
            <h1>Hello, world!</h1>
            <p> 연습장에 오신 걸 환영합니다! </p>
        </div>
    </div>

    <form role="form" action="login.php" method="POST">

        <div class="form-group">
            <label for="id"> 아이디 </label>
            <input type="text" class="form-control" name="id" placeholder="id">
        </div>

        <div class="form-group">
            <label for="pw"> 비밀번호 </label>
            <input type="password" class="form-control" name="pw" placeholder="pw">
        </div>

        <div class="form-group text-right">
            <button type="submit" class="btn btn-success"> login </button>
        </div>

    </form>

</div>
</body>
<script src="/mbd/include/js/bootstrap.min.js"></script>
</html>
